apps/source-storybook/src/components.stories.mdx

```diff
@@ -0,0 +1,9 @@
+import { Meta } from '@storybook/addon-docs';
+import ReadMe from '../../../libs/@guardian/source-react-components/README.md';
```

Comment:
I think I've found a way for us to import this using the package name.

- This line changed to: `import ReadMe from '@guardian/source-react-components/README.md';` 
- A package.json containing the following:
```json
{
	"name": "source-storybook",
	"devDependencies": {
		"@guardian/source-react-components": "workspace:*"
	}
}
```

works well for me locally 🙂

Reply:
hopefully the same for the other place we do this too

Reply:
Yeah thats nicer

Reply:
Great - you happy to commit those changes @ob6160 ?

Reply:
sure I’ll push it up in a bit